tests/HelpersTest.php

```diff
@@ -149,6 +149,22 @@ function stopwatch()
         $this->assertEquals(0.01, round($time, 2));
     }
 
+    /** @test */
+    public function toAssoc()
+    {
+        $ordered = collect([
+            ['name', 'John'],
+            ['lastName', 'Doe'],
+        ])->toAssoc();
+
+        $this->assertTrue(
```

Comment:
How about using the `assertSame` to assert the expected value is same as `0` value?